Services/DataX.Flow/Flow.ManagementService/Properties/launchSettings.json

```diff
@@ -20,8 +20,9 @@
     "Flow.ManagementService": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "api/values",
```

Comment:
why are we changing this, is this compat with the previous version?

Reply:
The change is compatible. Though, since the change is not required, reverting it.